src/Symfony/Component/Serializer/Mapping/AttributeMetadata.php

```diff
@@ -44,12 +44,21 @@ class AttributeMetadata implements AttributeMetadataInterface
     /**
      * @var string|null
      *
+     * @deprecated since Symfony 5.1, use $serializedNames instead.
```

Comment:
It shouldn't be `since Symfony 5.2` instead? It's the next release.

Reply:
done